grafana/data_source_organization_users_test.go

```diff
@@ -0,0 +1,61 @@
+package grafana
+
+import (
+	"fmt"
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatasourceOrganizationUsers(t *testing.T) {
+	CheckOSSTestsEnabled(t)
+
+	var organization gapi.Org
+	orgName := "datasource-org-users-test-org"
+	email := "foo@example.com"
+	checks := []resource.TestCheckFunc{
+		testAccOrganizationCheckExists("grafana_organization.test", &organization),
+		resource.TestCheckResourceAttr(
+			"data.grafana_organization_users.from_name", "organization_name", orgName,
+		),
+		resource.TestCheckResourceAttr(
+			"data.grafana_organization_users.from_name", "user_emails.#", "2",
+		),
+		resource.TestCheckResourceAttr(
+			// Grafana automatically adds the admin user to all orgs
+			"data.grafana_organization_users.from_name", "user_emails.0", "admin@localhost",
+		),
+		resource.TestCheckResourceAttr(
+			"data.grafana_organization_users.from_name", "user_emails.1", email,
+		),
+	}
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccOrganizationCheckDestroy(&organization),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatasourceOrganizationUsersConfig(orgName, email),
+				Check:  resource.ComposeTestCheckFunc(checks...),
+			},
+		},
+	})
+}
+
+func testAccDatasourceOrganizationUsersConfig(orgName, email string) string {
```

Comment:
I opted to use this configuration in tests -- rather than the `examples/data-sources/grafana_organization_users/data-source.tf` configuration -- so we can dynamically declare the `foo@example.com` user email address from within the tests, rather than needing to hard-code the expected email addresses across both the `examples/data-sources/grafana_organization_users/data-source.tf` and `grafana/data_source_organization_users_test.go` files.

Do others agree with this decision, or would you prefer I abandon `testAccDatasourceOrganizationUsersConfig` in favor of using a `examples/data-sources/grafana_organization_users/data-source.tf` configuration?